posts/2020/02/whats-next-for-pydatasparse.md

```diff
@@ -0,0 +1,37 @@
+<!--
+.. title: What's next for PyData/Sparse
+.. slug: whats-next-for-pydatasparse
+.. date: 2020-02-17 05:11:42 UTC-06:00
+.. tags: Sparse, SciPy, NumPy, Tensors, Big data
+.. type: text
+.. author: Hameer Abbasi
+-->
+
+# What have we been doing so far? 🤔
+## Research 📚
+A lot of behind the scenes work has been taking place on PyData/Sparse. Not so much in terms of code, more in terms of research and community/team building. I've decided more-or-less to use the structure and the research behind the [Tensor Algebra Compiler](https://github.com/tensor-compiler/taco), the work of Fredrik Kjolstad and his collaborators at MIT. 🙇🏻‍♂️ To this end, I've read/watched the following talks and papers:
```

Comment:
minor textual: I think "I've decided more-or-less" should be "I've more-or-less decided"

Reply:
Done!